core/src/Revolution/Processors/Security/ResourceGroup/GetNodes.php

```diff
@@ -85,13 +85,14 @@ public function process()
                     $resources = $resourceGroup->getResources();
                     /** @var modResource $resource */
                     foreach ($resources as $resource) {
+                        $icons = $resource->getIconClasses();
```

Comment:
This and the next change: Another super small thing, but let's make consistent (no need for var declaration)
```suggestion
```